app/(app)/(configure)/summary/page.tsx

```diff
@@ -206,7 +206,7 @@ export default function Summary() {
                 .catch((error: any) => {
                   // ignore 4001 "user rejected request" error code
                   if (error.code !== 4001) {
-                    alert(`Error ${error.code}: ${error.message}`);
+                    // MetaMask will show its own error message
```

Comment:
I'm kinda scared about this removing _all_ error alerting.

Can you confirm that the errors still show? Can you demonstrate this too, like with video or screenshots? That would make me feel more confident about this not having unintended side-effects 🙏